xstream/src/java/com/thoughtworks/xstream/io/xml/PrettyPrintWriter.java

```diff
@@ -238,7 +236,7 @@ private void writeText(final String text, final boolean isAttribute) {
             case '\t':
             case '\n':
                 if (!isAttribute) {
-                    writer.write(c);
+                    writer.write(Character.toChars(c));
```

Comment:
Right, I just meant in this case we know the character will be a single `char`. Not important.

Reply:
Right, and I knew that when deciding to use `Character.toChars(c)` in this case and the case below rather than prematurely optimizing by casting the `int` to a `char`.

This _review comment_ was unnecessary in this case I think.